modules/s3/outputs.tf

```diff
@@ -3,5 +3,5 @@ output "config_bucket_id" {
 }
 
 output "s3_bucket_id" {
-  value = aws_s3_bucket.model_buckets[2].id
+  value = aws_s3_bucket.model_buckets[0].id
```

Comment:
Wondering if this should be 1 instead of 0, cos based on what we discussed the python files and python files should be pushed to the config bucket which is at index 1 (in the s3 code above).

Reply:
What should the index of the one above be?

Reply:
The issue was with where this bucket is referenced in sagemaker.  The output would have been mismatched. I fixed it in the sagemaker module and tested it already.

Reply:
Thanks. Legend